src/openms/source/COMPARISON/SPECTRA/SpectrumPrecursorComparator.cpp

```diff
@@ -78,10 +78,14 @@ namespace OpenMS
 
     double mz1 = 0.0;
     if (!x.getPrecursors().empty())
+    {
       mz1 = x.getPrecursors()[0].getMZ();
+    }
     double mz2 = 0.0;
     if (!y.getPrecursors().empty())
+    {
```

Comment:
do you do this manually? Because I think a proper way would be to do clang format once